### DISCUSSION TITLE
充值后有充值记录，但余额是零，如何处理好？

### DISCUSSION THREAD

**Discussion by dgzbk790119** [ZuodaoTech/everyone-can-use-english#1027] [General]
充值后有充值记录，但余额是零，如何处理好？

**Comment by an-lee** [comment]
刚才有个服务挂了。现在应该好了。

**Comment by dgzbk790119** [comment]
谢谢








 ----------回复的邮件信息----------
 ***@***.***&gt; 于2024年8月30日(周五) 晚上8:36写道:

  

 
刚才有个服务挂了。现在应该好了。
